tutorials/Numerics/TimeStepping/explicit_lsrk.jl

```diff
@@ -0,0 +1,190 @@
+# # [Single-rate Explicit Timestepping](@id Single-rate-Explicit-Timestepping)
+
+# In this tutorial, we shall explore the use of explicit Runge-Kutta
+# methods for the solution of nonautonomous (or non time-invariant) equations.
+# For our model problem, we shall reuse the rising thermal bubble
+# tutorial. See its [tutorial page](@ref Rising-Thermal-Bubble-Configuration)
+# for details on the model and parameters. For the purposes of this tutorial,
+# we will only run the experiment for a total of 100 simulation seconds.
+
+using ClimateMachine
+const clima_dir = dirname(dirname(pathof(ClimateMachine)));
+include(joinpath(
+    clima_dir,
+    "tutorials",
+    "Numerics",
+    "TimeStepping",
+    "tutorial_risingbubble_config.jl",
+))
+
+FT = Float64;
+
+# After discretizing the spatial terms in the equation, the semi-discretization
+# of the governing equations have the form:
+
+# ``
+# \begin{aligned}
+#     \frac{\mathrm{d} \boldsymbol{q}}{ \mathrm{d} t} &= M^{-1}\left(M S +
+#     D^{T} M (F^{adv} + F^{visc}) + \sum_{f=1}^{N_f} L^T M_f(\widehat{F}^{adv} + \widehat{F}^{visc})
+#     \right) \equiv \mathcal{T}(\boldsymbol{q}).
+# \end{aligned}
+# ``
+
+# Referencing the canonical form introduced in [`Time integration`](@ref
+# Time-integration) we have that in any explicit
+# formulation ``\mathcal{F}(t, \boldsymbol{q}) \equiv 0`` and, in this particular
+# forumlation, ``\mathcal{T}(t, \boldsymbol{q}) \equiv \mathcal{G}(t, \boldsymbol{q})``.
+
+# The time step restriction for an explicit method must satisfy the stable
+# [Courant number](https://en.wikipedia.org/wiki/Courant%E2%80%93Friedrichs%E2%80%93Lewy_condition)
+# for the specific time-integrator and must be selected from the following
+# constraints
+#
+# ``
+# \Delta t_{\mathrm{explicit}} = min \left( \frac{C \Delta x_i}{u_i + a}, \frac{C \Delta x_i^2}{\nu} \right)
+# ``
+#
+# where ``C`` is the stable Courant number, ``u_i`` denotes the velocity components,
+# ``a`` the speed of sound, ``\Delta x_i`` the grid spacing (non-uniform in case of
+# spectral element methods) along the direction ``(x_1,x_2,x_3)``, and ``\nu`` the
+# kinematic viscosity. The first term on the right is the time step condition
+# due to the non-dissipative components, while the second term to the dissipation.
+# For explicit time-integrators, we have to find the minimum time step that
+# satisfies this condition along all three spatial directions.
```

Comment:
Does it mean to specify the CFL_direction to be EveryDirection? If so, it will be helpful to put a note here.